common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BrokerOperationBundle.java

```diff
@@ -97,11 +97,25 @@ public BrokerOperationBundle(@NonNull final Operation operation,
                                  @Nullable final Bundle bundle) {
         this.operation = operation;
         this.targetBrokerAppPackageName = targetBrokerAppPackageName;
-        this.bundle = bundle != null ? bundle : new Bundle();
+        this.bundle = deepCopy(bundle);
         this.bundle.putBoolean(KEY_REQUEST_ACTIVE_BROKER_DATA, true);
         Logger.info(TAG, "Requested Active Broker Data");
     }
 
+    // To be replaced by bundle.deepCopy() (requires API 26)
+    @NonNull
+    Bundle deepCopy(@Nullable final Bundle source) {
```

Comment:
can this be a private method?

Reply:
I'll move this out to a separate util class + add test.

Reply:
there's actually a better way to do this (sourceBundle.clone(), or new Bundle(sourceBundle))